about.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet" />
+    <title>CREATIVE CITY SUMMIT</title>
+</head>
+
+<body>
+    <header>
+        <nav class="upper-nav">
+            <ul class="upper-ul">
+                <li><a href="#"><img src="./images/facebook.png" alt="facebookIcon"></a></li>
+                <li><a href="#"><img src="./images/twitter.png" alt="twitter"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="https://www.google.com/">Log out</a></li>
+            </ul>
+        </nav>
+        <nav class="sec-navbar">
+            <span class="menu" id="menu"><i class="fas fa-bars"></i></span>
+            <a href="index.html"><img class="logo" src="./images/logo.png" alt="logo"></a>
+            <ul class="second-list" id="mobile-menu">
+                <li><i class="fas fa-times close-btn" id="close-btn"></i></li>
+                <li><a href="index.html" class="nav-list" id="about-link">Home</a></li>
+                <li><a href="#" class="nav-list">Program</a></li>
+                <li><a href="#" class="nav-list">Join</a></li>
+                <li><a href="#" class="nav-list">Sponsor</a></li>
+                <li><a href="#" class="button nav-list">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    </main>
+    <main class="main-container about-container">
+        <div class="center-about">
+            <h1 class="top-h1-about">Hello! sharing world</h1>
+            <h2 class="main-sec-heading-about">CREATIVE CITY SUMMIT 2015</h1>
+            <p class="main-par-about">
+                Connecting with sector colleagues is an essential part of developing culture in Canada. Every year, delegates representing culture work across the country gather to share, learn, and grow..
+                Connecting with sector colleagues is an essential part of developing culture in Canada. Every year, delegates representing culture work across the country gather to share, learn, and grow..
+            </p>
+            <p class="contact-us">
+                Do you have other questions about the CC Global Summit 2015! </p>
+            <a href="#" class="email-id">ccsummit2015@cccanada.org </a>
+        </div>
+    </main>
```

Comment:
- [ ] Please use `section` tag instead of `main` tag for your hero section of the app or you can inclose all your `section` tags inside the `main` tag. 

The` <main>` tag specifies the main content of a document. The content inside the `<main>` element should be unique to the document. It should not contain any content that is repeated across documents such as sidebars, navigation links, copyright information, site logos, and search forms. `<section>` element is used to represent a group of related content. You can use `<section>` inside the `<main>` tag.